### ISSUE TITLE
Remove/change the default password requirements

### ISSUE THREAD

**Issue by domsson** [getgrav/grav#1594]
After installing Grav+Admin and opening the site, you are asked to create a first user. Here, Grav *enforces* specific password requirements (numbers, uppercase and lowercase letters, min length 8).

I understand that this is for the sake of the user, but as an informed user, [I want to be able to choose whatever password I please](https://blog.codinghorror.com/password-rules-are-bullshit/#1passwordrulesarebullshit), and be it [`correcthorsebatterystaple`](https://xkcd.com/936/). Surely, password length in particular might be a really useful requirement and one could argue that it should be fine to reduce the requirements to that, but I say let's simply change *requirement* to *suggestion*.

In other words, I would love to see Grav allow whatever password I want to use, but warn me if the password is weak according to whatever rules.

Discuss.

**Comment by paulmassen** [comment]
Before creating an user, you can adapt the password requirements to your need by editing system.yaml line 7 and adding your own regex expression

**Comment by domsson** [comment]
@paulmassen, that's very good to know, thank you for pointing that out! In that case, I guess, I'm suggesting to change the default regex. However, seeing that it can be configured rather easily (given sufficient knowledge of regex), it doesn't seem as important anymore. Still...

**Comment by rhukster** [comment]
I have a note to document these password requirements, but I think for the sake of security these default requirements are a good place to start.

**Comment by philippejadin** [comment]
Sorry to re-open such an old issue, but I think the problem here is that the first user creation form doesn't tell explicitly what makes a good password according to Grav. 

It could be explained in the password form "Please choose a strong password with at least lower and uppercase letters, and numbers".

Just got bitten by this, as a first time tester I almost gave up after 3 attempts to write a good enough password. This is mainly a problem for adoption by new users, like me, trying to test it locally, who don't want to be bothered more that a few minutes to test a new tool. I hope that make sense, and of course, thank you for the tool that looks really really good!
